quinn-proto/src/transport_parameters.rs

```diff
@@ -25,6 +25,41 @@ use crate::{
     RESET_TOKEN_SIZE, TIMER_GRANULARITY,
 };
 
+#[repr(u32)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub(crate) enum TransportParameterId {
+    // https://www.rfc-editor.org/rfc/rfc9000.html#iana-tp-table
+    OriginalDestinationConnectionId = 0x00,
+    MaxIdleTimeout = 0x01,
+    StatelessResetToken = 0x02,
+    MaxUdpPayloadSize = 0x03,
+    InitialMaxData = 0x04,
+    InitialMaxStreamDataBidiLocal = 0x05,
+    InitialMaxStreamDataBidiRemote = 0x06,
+    InitialMaxStreamDataUni = 0x07,
+    InitialMaxStreamsBidi = 0x08,
+    InitialMaxStreamsUni = 0x09,
+    AckDelayExponent = 0x0A,
+    MaxAckDelay = 0x0B,
+    DisableActiveMigration = 0x0C,
+    PreferredAddress = 0x0D,
+    ActiveConnectionIdLimit = 0x0E,
+    InitialSourceConnectionId = 0x0F,
+    RetrySourceConnectionId = 0x10,
+
+    // Smallest possible ID of reserved transport parameter https://datatracker.ietf.org/doc/html/rfc9000#section-22.3
+    ReservedTransportParameter = 0x1B,
+
+    // https://www.rfc-editor.org/rfc/rfc9221.html#section-3
+    MaxDatagramFrameSize = 0x20,
+
+    // https://www.rfc-editor.org/rfc/rfc9287.html#section-3
+    GreaseQuicBit = 0x2AB2,
+
+    // https://datatracker.ietf.org/doc/html/draft-ietf-quic-ack-frequency#section-10.1
+    MinAckDelayDraftIetfQuicAckFrequency07 = 0xFF04DE1B,
```

Comment:
nit: This seems much more verbose than necessary.
```suggestion
    MinAckDelayDraft07 = 0xFF04DE1B,
```